json-rpc/src/tests/unit_tests.rs

```diff
@@ -240,15 +242,15 @@ fn test_transaction_submission() {
 
 // TODO: Once account configs are published in the mock DB this test can be turned back on
```

Comment:
Can this be uncommented?

Reply:
this test requires mocking the on-chain currency resources, which is a much larger task.. for now we can just manually curl